upstream-community-operators/rqlite-operator/0.0.1/rqlite-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,202 @@
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          verbs:
+          - get
+        - apiGroups:
+          - rqcluster.example.com
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: rqlite-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  maturity: alpha
+  minKubeVersion: 1.12.0
+  keywords: []
+  labels: {}
+  selector:
+    matchLabels: {}
+  provider:
+    name: Jeff McCormick
+  maintainers:
+    - name: Jeff McCormick
+      email: djm76262@gmail.com
+  links:
+    - name: github repository
+      url: 'https://github.com/jmccormick2001/rqlite-operator'
+  icon:
+    - base64data: >-
+        iVBORw0KGgoAAAANSUhEUgAAAIMAAAB+CAYAAAAHkaKhAAAABmJLR0QA/wD/AP+gvaeTAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH4AQbASMgLA364gAABWlJREFUeNrt3c9PXFUYxvFnDszAtDNhbBkqqf0BNBVTEwuxEBe0YNKwMVZr4kLd+A+oMUZafyytsjO6swvXutESEwFN1IVWMSYuSlAjLYImFkwsggwzlLkuDKaYGJm599zzvuc+3w2ruRyYT957584ZABhjjDHGGGOMMcYYY4wxxhhjzGUpLQudvDwfXHx/Zsc/VK5jP1Yampyu+fnT7ThxOKfmd9yoYZEffv5TcObZcaxXNnf8mKamBWTuPuoUxMczy5i6thL0deRVgDAaIJx9bqImCABQLm+gcuUH5DfLztZ+o7SJ4Te/x9TcSsDThCMI2ydE2vmEKGQbMPHUneg7LHtCGJ8h/DMhpgVMiDfkT4iUzxC2TYjmNNLHjmLV8YT46Olu3HtI5kWlSQIEACivb2BDwIR4bGwVU9eDgBgcQbgVhPNTRgUY/qAqEoSRBOFhixC2gbhCEGIxbEEoW4Yg6mWnQBAmaRAIQigGVxAIQhgG1xBE3YcQAsIkGQJfZTjGIA0CQTjCIBUCQcSMQToEgogJgxYISQdhCIEgYsGgFUJSQRhCIAirGHyBkDQQkW+IHf9iPnh0ZBLVaoB0o5sbnAM97Xjywe7Ij9vUnMaFy6v49uc15yD69qUi3yCT8m0inOptx6cXH7K2k6hys/pa/+j0SL0gikcOYWlXMdQaChlg4gGDqEEYQqitTKM599XIsdHjd+zy7pRhCIEgIsOQNAg+gwiFYfLLheCsYwgnHUCQCOKbxfAgQmF4d/JH63sW/28ifOYIgjQQl+YCt5PBZaccTgSJIES9mkgqBJ9AGEIgCJUYJEPwAYQhBIJQhUETBM0gDCEQhAoMmiFoBNFICPGAqNyson90euQXYqitkx5B+DeI0++sjCz9IXPEiztN5Fpb8PgT/fCxTKM5N3AkH/lxe/YCj3R6hiHX2oJyVyc2ghTYziG81AukjUcYCMEtBDEYCKH2jkcMQQSGXGsLKoTgHIJzDFsQKoRQM4SMhWfOEAIhOMWQLxKCNAhOMOSLLSh3EkLNEHrsQogdAyGEgNBg/3sZQiCEWDEQQu3dEzOEWDAQQn0QXo4ZgnUMhKAHglUMhKALgjUMhKAPghUMhKATQuQYCEEvhEgxEIJuCJFhIIQ6IOyRBSESDIRQJ4ReWRCAkLujd7e2oNyZjRxCEPgL4a5CgB6BEEJPhj/zBSsT4a2ZAFeX/SSxt0kmBGv3GcK2sAoMjVW9AzE+txlsVOWuT+zH6+Y9AyEdgmgMPoHQAEE8Bh9AaIGgAoNmEJogqMGgEYQ2CKow3ApiVjgIjRDUYdgCcb9gEFohqMQgGYRmCGoxSAShHUJoDOeH23HgtkziQbiGkE8D3XtMyimGrrZs6pNnuhMNQgKEwQMNKeeTIekgfIIQ2TVDEkG4hpCLGEKkF5BJAiEBwlDEEICI/3sdAMwuloKh17/Dwu+Vuo9R6DiIG/m2uh9/MAe8PZhCMRv9E7G4FsAlhN1pYGB/g5VtZVYOGhZEWAwAcHsWeKUPKDbDm2xNBKv3GSScMn4tAS9OAb+tE4JTDJJAnJ8CFkuE4BSDFBDXS8ALX+sFERcE6xikgFhUCiJOCLFgIAgdEGLDQBC1lXcAIVYMBLFzCIMOIMSOgSDkQnCCgSBkQnCGgSDkQXCKgSBkQXCOIekgJEEQgSGpIKRBEIMhaSAkQgAsvYUdptmlUnDmvTVMbxacraEtC7xq6e1vqRBEYgCA2eUgGBqrYmEVTkFcOPH31yRAEIvBRxDSIYjG4BMIDRDEY/ABhBYIKjBoBqEJghoMWyAuXXP7scq2bAr37dv5GroKhn8ckzHGGGOMMcYYY4wxxhhj7D/6C6s6jhtV+7AcAAAAAElFTkSuQmCC
+      mediatype: image/png
+  replaces: rqlite-operator.v0.0.0
```

Comment:
@jmccormick2001, please consider removing this line as rqlite-operator.v0.0.0 doesn`t exist on operaturhub.io

Reply:
good catch, removed it just now.